Sources/BluetoothHostController.swift

```diff
@@ -29,6 +29,9 @@ public protocol BluetoothHostControllerInterface {
     /// Send a command to the controller and wait for response.
     func deviceRequest<C: HCICommand>(_ command: C, timeout: HCICommandTimeout) throws
     
+    /// Send a command to the controller and wait for response.
+    func deviceRequest<C: HCICommand, EP: HCIEventParameter>(_ command: C, _ eventParameterType: EP.Type, timeout: HCICommandTimeout) throws -> EP
```

Comment:
@jmarkstar  Open a PR in BluetoothLinux and BluetoothDarwin implementing this method. Also `TestController` must be updated.